Changelog

```diff
@@ -10,11 +11,13 @@ Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v5.1.0
    - BUGFIX: Fix filesize sorting in preview dialog (DoubleSpicy) c080fc3aa09400d7bfdecf73ad5fb409e02a8110
    - BUGFIX: Improve the speed icons in the status bar (Mahdi Hosseinzadeh) 928de360931579304128a8203a8ac61e4f32bed8
    - BUGFIX: Update link to news (tinyboxvk) 61ff683f11a653fecd6a3dd0dbfe784cd6aaf63d
+   - WEBUI: Improvements that should help with assistive technologies (Chocobo1)
+   - WEBUI: Internal refactoring to migrate away from mootools and towards native browser APIs
```

Comment:
Is it missing author?

Reply:
No? AFAIK a lot of authors are responsible for this generic entry.
If you can provide names for attribution I'll add them.

Reply:
IIRC it is mostly me and skomerko contributed to it.
https://github.com/qbittorrent/qBittorrent/pulls?q=is%3Apr+is%3Aclosed+label%3AWebUI+milestone%3A5.1+label%3A%22Code+cleanup%22

Reply:
@sledgehammer999 
BTW, "mootools" should be written as "MooTools".